### PR TITLE
contract change: added customProperties to ApiServiceProperties

### PR DESCRIPTION
**Pull request by ailn**
https://github.com/Azure/hydra-specs-pr/pull/920

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ailn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ailn). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ApiManagementManagement/packages.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
-  <package id="Microsoft.Azure.Common" version="2.0.3" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.0.4" targetFramework="net45" />
```
Comment (markcowl) [inline src/ApiManagementManagement/packages.config]: If you are going out with the //build or ignite release, you will need to update this to 2.1.0

**Comment by markcowl** [comment]
@ailn Closing, as merge conflicts need to be fixed and the common version needs to be updated to 2.1.0.  Please reopen or ope a new PR when these issues are fixed

**Comment by azurecla** [comment]
Hi __@ailn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ailn). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ComputeManagement/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Virtual Machines and Hosted Services.")]
 
-[assembly: AssemblyVersion("11.0.0.0")]
-[assembly: AssemblyFileVersion("11.2.1.0")]
+[assembly: AssemblyVersion("12.0.0.0")]
```
Comment (markcowl) [inline src/ComputeManagement/Properties/AssemblyInfo.cs]: revert this change

File: src/SiteRecoveryManagement/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.5.1.0")]
+[assembly: AssemblyFileVersion("0.6.0.0")]
```
Comment (markcowl) [inline src/SiteRecoveryManagement/Properties/AssemblyInfo.cs]: revert this change

File: src/StorSimple/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -21,7 +21,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure StorSimple management functions")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.2.0")]
```
Comment (markcowl) [inline src/StorSimple/Properties/AssemblyInfo.cs]: revert this change

**Comment by markcowl** [comment]
@ailn some changes in other partners assemblyinfo files that need to be reverted.

**Comment by ailn** [comment]
Oh, sorry for that
